README.md

```diff
@@ -55,7 +55,7 @@
 <tr>
   <td>Downloads</td>
   <td>
-    <a href="https://pandas.pydata.org">
+    <a href="https://zarr-developers.github.io">
```

Comment:
Thanks for catching this and for the fix @andersy005! Would you mind updating the link to point to the zarr docs instead of the blog

```suggestion
    <a href="https://zarr.readthedocs.io">
```

Reply:
Sounds good! Done